pocs/state/states/default/parked.py

```diff
@@ -4,24 +4,26 @@ def on_enter(event_data):
     pocs = event_data.model
     pocs.say("I'm parked now. Phew.")
 
-    has_valid_observations = pocs.observatory.scheduler.has_valid_observations
+    if pocs.run_once is True:
+        pocs.say("Done running loop, going to clean up and sleep!")
+        pocs.next_state = 'housekeeping'
+    elif pocs.should_retry is False:
+        pocs.say("Done with retrying loop, going to clean up and sleep!")
+        pocs.next_state = 'housekeeping'
+    else:
+        has_valid_observations = pocs.observatory.scheduler.has_valid_observations
 
-    if has_valid_observations:
-        if pocs.is_safe():
-            if pocs.should_retry is False or pocs.run_once is True:
-                pocs.say("Done retrying for this run, going to clean up and shut down!")
-                pocs.next_state = 'housekeeping'
-            else:  # This branch will only happen if there is an error causing a shutdown
+        if has_valid_observations:
```

Comment:
I take it that `has_valid_observations` being True means that those observations could be taken now, so it isn't yet Astronomical dawn, or some such.

Reply:
It doesn't capture the logic of whether those observations are observable or not, it just captures whether or not there is even something to try to schedule. This mostly comes into play if targets are removed while running. Not a widely used feature right now in terms of actively adding/removing fields while running.